agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1650,7 +1681,8 @@ bool backhaul_manager::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_r
         // Handle the CMDU message. If the message was processed locally
         // (by the Backhaul Manager), this function will return 'true'.
         // Otherwise, it should be forwarded to the slaves.
-        if (handle_1905_1_message(cmdu_rx, src_mac)) {
+        Socket *destinationAP = nullptr;
```

Comment:
```suggestion
        Socket *target_slave = nullptr;
```

**BUT**, I think it's not required at all.
You can just forward the CMDU from your handler function to the correct slave and return `true` (so it won't be forwarded to all slaves here).